app/controllers/application_controller.rb

```diff
@@ -1,8 +1,10 @@
 class ApplicationController < ActionController::Base
   include Slimmer::Template
+  include NewBrowseAbTestable
 
-  helper_method :level_two_browse_variant
```

Comment:
This :level_two_browse_variant was cruft and should have been removed a looooong time ago